src/cpp/handler/httpsession.cpp

```diff
@@ -210,10 +212,10 @@ class HttpSession::Private : public QObject
 		errorConnection = req->error.connect(boost::bind(&Private::req_error, this));
 
 		timer = new RTimer(this);
-		connect(timer, &RTimer::timeout, this, &Private::timer_timeout);
+		timeoutConnection = timer->timeout.connect(boost::bind(&Private::timer_timeout, this));
 
 		retryTimer = new RTimer(this);
-		connect(retryTimer, &RTimer::timeout, this, &Private::retryTimer_timeout);
+		retryTimerConneciton = retryTimer->timeout.connect(boost::bind(&Private::retryTimer_timeout, this));
 		retryTimer->setSingleShot(true);
```

Comment:
The original code has explicit disconnects for these too so we'll need to retain that behavior.

Reply:
updated to boost disconnect